pages/countries/[slug].js

```diff
@@ -25,6 +27,12 @@ export default function SpotDetails({ surfspots }) {
       </Head>
       <Header onMapShown={handleMapShown} />
       <ListSurfspots>
+        <StyledSection>
+          <StyledLink href={"/countries"}>
+            <Image src={backButton} alt="backButton" width={25} height={25} />
```

Comment:
du mischst in deinen Alt-Attributen Deutsch und Englisch... Ist jetzt nicht so wild, würde ich aber glaube ich einheitlich machen...